config/routes.rb

```diff
@@ -1,56 +1,15 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
+  root 'users#show'
 
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  post "/auth/developer/callback", to: "sessions#create"
+  get "/auth/:provider/callback", to: "sessions#create"
+  get "/search" => "searches#search", as: :search
+  delete "/logout" => "sessions#destroy", as: :logout
+  get '/feed', to: "users#show", as: :feed
+  post '/follow', to:"creators#follow", as: :follow
+  post '/unfollow', to:"creators#unfollow", as: :unfollow
 
-  # Example of regular route:
-  #   get 'products/:id' => 'catalog#view'
+  resources :sessions, except: [:update, :destroy]
+  resources :creators
 
-  # Example of named route that can be invoked with purchase_url(id: product.id)
-  #   get 'products/:id/purchase' => 'catalog#purchase', as: :purchase
-
-  # Example resource route (maps HTTP verbs to controller actions automatically):
-  #   resources :products
-
-  # Example resource route with options:
-  #   resources :products do
-  #     member do
-  #       get 'short'
-  #       post 'toggle'
-  #     end
-  #
-  #     collection do
-  #       get 'sold'
-  #     end
-  #   end
-
-  # Example resource route with sub-resources:
-  #   resources :products do
-  #     resources :comments, :sales
-  #     resource :seller
-  #   end
-
-  # Example resource route with more complex sub-resources:
-  #   resources :products do
-  #     resources :comments
-  #     resources :sales do
-  #       get 'recent', on: :collection
-  #     end
-  #   end
-
-  # Example resource route with concerns:
-  #   concern :toggleable do
-  #     post 'toggle'
-  #   end
-  #   resources :posts, concerns: :toggleable
-  #   resources :photos, concerns: :toggleable
-
-  # Example resource route within a namespace:
-  #   namespace :admin do
-  #     # Directs /admin/products/* to Admin::ProductsController
-  #     # (app/controllers/admin/products_controller.rb)
-  #     resources :products
-  #   end
```

Comment:
To reduce PR size, go ahead and delete the default comments that you're not using throughout the Rails app.